doubleml/double_ml.py

```diff
@@ -354,6 +354,7 @@ def psi(self):
         Values of the score function after calling :meth:`fit`;
         For models (e.g., PLR, IRM, PLIV, IIVM) with linear score (in the parameter)
         :math:`\\psi(W; \\theta, \\eta) = \\psi_a(W; \\eta) \\theta + \\psi_b(W; \\eta)`.
+        The shape is ``(n_obs, n_rep, n_coefs)``.
```

Comment:
Thanks for all the changes. I thought `_predictions` was a dictionary with keys corresponding to the learner names, so the shape info rather refers to the corresponding values?

Reply:
Great point. I will fix this.